File: janus/janus-extras/modules/scriptedagent/jrubyengine/src/test/resources/org/janusproject/jrubyengine/agent2.rb
def activateAgent(agent)
	agent.scriptedActivateExecuted = true;
end

def liveAgent(agent)
	if (agent.scriptedLiveExecuted) then
		agent.killMe()
		agent.scriptedKilledExecuted = true;
	else
		agent.scriptedLiveExecuted = true;
	end
end

def endAgent(agent)
	agent.scriptedEndExecuted = true;
end

File: janus/janus-extras/modules/scriptedagent/luaengine/src/test/resources/org/janusproject/luaengine/agent2.lua
function activateAgent(agent)
	agent.scriptedActivateExecuted = true
end

function liveAgent(agent)
	if (agent.scriptedLiveExecuted) then
		agent:killMe()
		agent.scriptedKilledExecuted = true
	else
		agent.scriptedLiveExecuted = true
	end
end

function endAgent(agent)
	agent.scriptedEndExecuted = true
end
